pkg/apis/numaplane/v1alpha1/progressive_status_types.go

```diff
@@ -90,12 +90,12 @@ type PromotedChildStatus struct {
 type PromotedPipelineTypeStatus struct {
 	PromotedChildStatus `json:",inline"`
 
-	// ScaleValues is a map where the keys are the promoted child source vertices names
-	// and the values are the scale values of the source vertices
+	// ScaleValues is a map where the keys are the promoted child vertices names
+	// and the values are the scale values of the vertices
 	ScaleValues map[string]ScaleValues `json:"scaleValues,omitempty"`
-	// AllSourceVerticesScaledDown indicates if ALL the promoted child source vertices have been scaled down
-	AllSourceVerticesScaledDown bool `json:"allSourceVerticesScaledDown,omitempty"`
-	// ScaleValuesRestoredToOriginal indicates if ALL the promoted child source vertices have been set back to the original min and max scale values.
+	// AllVerticesScaledDown indicates if ALL the promoted child vertices have been scaled down
+	AllVerticesScaledDown bool `json:"allVerticesScaledDown,omitempty"`
```

Comment:
we no longer treat "Source" vertices different from other vertices. All vertices need to be scaled down in the promoted pipeline during the upgrade